### ISSUE TITLE
Error when visiting /settings/applications

### ISSUE THREAD

**Issue by der-On** [pixelfed/pixelfed#2231]
With `OAUTH_ENABLED=true` I was previously able to manage private access tokens when visiting `/settings/applications`

Since latest update I get the following error with `GET /oauth/tokens`

```
Target class [Barryvdh\\Cors\\HandleCors] does not exist. at /var/www/vendor/laravel/framework/src/Illuminate/Container/Container.php:807
```

**Comment by der-On** [comment]
A quick look into the file under `vendor/barryvdh/cors/HandleCors.php` shows me the following:

```php
namespace Fruitcake\Cors;
```

Seems the namespace for the module has changed, but the path has not.

**Comment by dansup** [comment]
@der-On Which branch and commit are you on?

**Comment by der-On** [comment]
@dansup I'm on staging. commit https://github.com/pixelfed/pixelfed/commit/6e0cb9dca8ef28023bdf7d786c77dcfa6d90443a

**Comment by der-On** [comment]
@dansup So it seems the author just moved the entire package to a new home https://github.com/fruitcake/laravel-cors. I tried to fix this here: https://github.com/pixelfed/pixelfed/pull/2236
Seems to work on my install now.

**Comment by der-On** [comment]
@dansup My PR allows to edit api tokens, but the api now still does not work. I will investigate and hopefully get a fix for it into my PR.

Edit: It was my mistake. I've used the wrong URL to my pixelfed instance. So I can happily report that my pixelfed instance is working well with the patch provided in the PR.

**Comment by dansup** [comment]
Fixed in #2220
